### DISCUSSION TITLE
Cheat Sheet/Guide?

### DISCUSSION THREAD

**Discussion by KremSH** [calebstewart/pwncat#175] [Q&A] [answered]
Hello ,

Sorry if this question was asked before or if there is an obvious resource for it but is there like a cheat sheet for pwncat , for example: as I am using it I dont know how to push a session into the background and go to the pwncat cli , among other useful shortcuts.

**Comment by Mitul16** [comment, marked as answer]
When you have a valid session, press `Ctrl+D` to switch between your _local `pwncat` prompt_ and _remote terminal prompt_

There are a lot of nicely documented instructions available on __Read the Docs__, [here](https://pwncat.readthedocs.io/en/latest/)
You may start with the [basic usage](https://pwncat.readthedocs.io/en/latest/usage.html)

<details>
  <summary><strong>pwncat</strong>'s local prompt</summary>

1. Get a session
2. You will be greeted with a _local `pwncat` prompt_
    ```bash
    # see available help
    (local) pwncat$ help
    .
    .
    .
    # run a complete system enumeration, `pwncat` will intelligently save the results for multiple uses
    (local) pwncat$ run enumerate.gather
    .
    .
    .
    ```
3. Interact with your remote machine (press `Ctrl-D` to switch)
4. Create another session if you want, and switch between them with ease
    ```bash
    # list available sessions
    (local) pwncat$ sessions
    .
    .
    .
    # interact with a session of your choice
    (local) pwncat$ sessions <SESSION-ID>
    .
    .
    .
    # happy hacking 😁
    ```
    __EDIT__: fixed the incorrect command ~~`session`~~ `sessions`
</details>

If you would like to view its practical usage from __Caleb Stewart__ himself along with __John Hammond__, there is a video available [here](https://www.youtube.com/watch?v=CISzI9klRkw)
> Note: this video is a little older, and the [latest documentation](https://pwncat.readthedocs.io/en/latest/) is where you will find the latest usage information and instructions

**Reply by KremSH** [reply]
Thank you for the response , I’ll be sure to check the documentation next time :)

**Reply by trevorbryant** [reply]
Feel free to PR any documentation updates you may see to recommend. The start of a cheatsheet could always be helpful as well. :smile:

**Reply by calebstewart** [reply]
I've been out due to DEFCON the past week. Thanks for answering this @Mitul16!

@KremSH if this answers your question, please mark Mitul's response as an answer so future users can find the answer as well :)

I intended the "basic usage" page of the documentation to be a sort of cheat sheet. Like @trevorbryant said, if you have anything you think would be useful there, please let us know or open a pull request if you're feeling up to it, and I'm happy to add it in!
